### PR TITLE
spruce up app page

### PR DESCRIPTION
**Pull request by ungoldman**
- nice looking hero section with title, subtitle, downloads
- use feature blocks
- better screenshots
- use 3 column view for featured torrents
- add more links to third party references

I wanted to note here that we're calling it `WebTorrent.app` in the repo and `WebTorrent Desktop` on the site (with path of `/app`). Would be good to pick one and stick with it in both places.

preview:

app-page

### CODE REVIEW

**Comment by feross** [comment]
Looks amazing!
For this app, here's what I think we should use:

- `WebTorrent Desktop` on the website and in GitHub issues – distinguishes it from WebTorrent CLI, the webtorrent npm package, and the WebTorrent protocol
- `WebTorrent` throughout the app itself. It would be redundant to write `WebTorrent Desktop` within the desktop app, IMO.

I just renamed the main repo to `webtorrent-desktop`.

**Comment by ungoldman** [comment]
Alright, I'll fix links in here to point to `webtorrent-desktop`

**Comment by feross** [comment]
@ungoldman Sounds good to me. I added you to the repo as collaborator. Go ahead and self-merge this once you've updated the links to `webtorrent-desktop`.

**Comment by feross** [comment]
Also note: Merges to this repo automatically deploy to https://webtorrent.io, so be careful :)

**Comment by ungoldman** [comment]
oh boy! I'll try to be careful :)

File: package.json [hunk 1]
```diff
@@ -43,7 +43,7 @@
     "throttleit": "^1.0.0",
     "unlimited": "^1.1.0",
     "webtorrent": "0.x",
-    "webtorrent-app": "feross/webtorrent-app",
+    "webtorrent-desktop": "git+ssh://git@github.com/feross/webtorrent-desktop.git",
```
Comment (ungoldman) [inline package.json]: I had to change this to a github url to get it to work locally. I'm a little concerned about the idea of requiring the desktop application (which runs electron-prebuilt postinstall) as a dependency in www. I guess it's working already but seems a little smelly to me.

Comment (feross) [inline package.json]: This probably stopped working when we renamed the repo. Looks like you changed it back and it works fine, right?

Comment (ungoldman) [inline package.json]: Yeah, it stopped working for `feross/webtorrent-app` AND `feross/webtorrent-desktop`, `git+ssh` worked locally but broke travis, now `feross/webtorrent-desktop` seems to be working again. shrug

Comment (feross) [inline package.json]: I think this is fine for the time being, but lmk if you think of a better way to handle getting the latest version.

**Comment by dcposch** [comment]
LGTM! nice design

File: package.json (hunk 1 above)
Comment (ungoldman) [inline package.json]: Might be able to just use the github releases API. That would give us access to latest version info and download assets.

**Comment by ungoldman** [comment]
Okay I'm pushing the button

File: package.json (hunk 1 above)
Comment (ungoldman) [inline package.json]: https://www.npmjs.com/package/ghreleases#getlatestauth-org-repo-options-cb

**Comment by ungoldman** [comment]
Weeee looks like I didn't break anything: https://webtorrent.io/desktop

**Pull request merged**